### ISSUE TITLE
Exact training setup

### ISSUE THREAD

**Issue by vobecant** [Banconxuan/RTM3D#30]
Dear authors,

thank you very much for providing us the code for the "KM3D" paper (is it code corresponding to KM3D and not to RTM3D, right?).

I would like to reproduce your results on the KITTI dataset and then apply your method to the Waymo Open dataset. Therefore, I would like to know your exact training setup. Namely, I am interested in:

- batch size
- number of GPUs
- learning rate and learning rate schedule
- What does the _right images augmentation_ mean?
- training images size

Thank you very much in advance!

Best,
Antonin.
